apps/src/code-studio/components/progress/UnitCalendarButton.jsx

```diff
@@ -64,3 +64,10 @@ export default class UnitCalendarButton extends React.Component {
     );
   }
 }
+
+const styles = {
```

Comment:
Just wonder if the size of the button adapts with changes in the length of the text. Thinking about translations.

Reply:
<img width="1055" alt="Screen Shot 2023-02-08 at 4 32 37 PM" src="https://user-images.githubusercontent.com/37230822/217683655-beb67dbe-8aed-4d8f-9307-4554599fbc9f.png">

Yep- just edited the text so you can see what it looks like!